pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -187,6 +217,28 @@ func configureFlexVolume(driverFile, driverDir, driverName string) error {
 		return fmt.Errorf("failed to rename %s to %s: %+v", destFile, finalDestFile, err)
 	}
 
+	// Write the flex configuration
+	enableSELinuxRelabeling, err := strconv.ParseBool(os.Getenv(agent.RookEnableSelinuxRelabelingEnv))
+	if err != nil {
+		logger.Errorf("invalid value for disabling SELinux relabeling. %+v", err)
+		enableSELinuxRelabeling = true
+	}
+	enableFSGroup, err := strconv.ParseBool(os.Getenv(agent.RookEnableFSGroupEnv))
+	if err != nil {
+		logger.Errorf("invalid value for disabling fs group. %+v", err)
+		enableFSGroup = true
+	}
+	settings, err := generateFlexSettings(enableSELinuxRelabeling, enableFSGroup)
+	if err != nil {
+		logger.Errorf("invalid flex settings. %+v", err)
+	} else {
+		if err := ioutil.WriteFile(path.Join(driverDir, settingsFilename), settings, 0644); err != nil {
```

Comment:
This "extra" file doesn't cause any issues for the kubelet?

Reply:
No, the kubelet looks for the driver name matching the directory name. I haven't seen any side effects, but I'm doing more testing now.

Reply:
@galexrt No messages in the kubelet log, and no other side effects. Everything looks good to me and the integration tests passed in the first run. I also added a unit test now.